client/app/components/Technologies/style.module.scss

```diff
@@ -0,0 +1,41 @@
+@import "@style/globalVariables.scss";
+
+.technologiesBar {
+    height: 300px;
+    background-color: #2E2E2E;
+    overflow: hidden; // Ensures the logos outside the container are not visible
+    position: relative;
+}
+
+.technologiesTitle {
+    color: white;
+    font-family: $RubikLight;
+    font-size: $font-size-medium;
+    margin-left: 100px;
+    padding-top: 20px;
+    padding-bottom: 20px;
+}
+
+.logos {
+    display: flex;
+    flex-direction: row;
+    animation: scrollLeft 30s linear infinite;
+    will-change: transform;
+}
+
+.logo {
+    width: 70px;
+    height: 70px;
+    margin-left: 35px;
+    margin-right: 35px;
```

Comment:
evita sa folosesti pixeli la margine, 
in general ar trebui folositi pixeli doar la dimensiuni de imagin i/logouri cum ai facut aici la width si height si maixm la padding. in rest fa cu em sau rem.